docs/build/build-client-side.md

```diff
@@ -0,0 +1,187 @@
+---
+id: build-client-side
+title: Building Client-side Apps
+sidebar_label: Building Client-side Apps
+description: An overview about building clients that use Polkadot
+keywords: [data, index, query, explorer, dashboard, dapp, uapp, app, frontend, client]
+slug: ../build-client-side
+---
+
+import Tabs from "@theme/Tabs"; import TabItem from "@theme/TabItem";
+
+As with any blockchain, building decentralized applications (dApps) is a significant part of how a
+developer can build on Polkadot. Within the ecosystem, you also hear the term "Unstoppable App", or
```

Comment:
I would add conditional rendering for Kusama. I would remove the term "unstoppable" as not appropriate from the regulatory point of view. better using resilience